extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml

```diff
@@ -23,16 +23,16 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>stratos-das-extension</artifactId>
         <groupId>org.apache.stratos</groupId>
+        <artifactId>stratos-das-extension</artifactId>
         <version>4.1.5-SNAPSHOT</version>
         <relativePath>../../../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>metering-service-capps</artifactId>
-    <name>Apache Stratos - Metering Service Composite Applications</name>
-    <description>Apache Stratos Metering Service Composite Applications</description>
+    <artifactId>org.apache.stratos.metering.service.capps</artifactId>
```

Comment:
Usually this pattern is followed if there are Java sources in the modules. Do we have any Java sources inside CApps?